portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,16 +63,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String enteredName = request.getParameter("name-entry");
     String enteredComment = request.getParameter("comment-entry");
-    Comment comment = new Comment(enteredName, enteredComment);
-    //String json = convertToJson(comment);
-    comments.add(enteredComment); //testing to se if it's working
+    long timestamp = System.currentTimeMillis();
```

Comment:
One of the alternatives is to simply get the value and pass to the setProperty() method:
commentEntity.setProperty("author", request.getParameter("name-entry"));

Saves three lines. But this way is fine as well :)